chunky/src/java/se/llbit/math/Grid.java

```diff
@@ -8,7 +8,7 @@
 import java.util.Random;
 
 public class Grid {
-  private static final int GRID_FORMAT_VERSION = 0;
+  private static final int GRID_FORMAT_VERSION = 1;
```

Comment:
Is the new format version still compatible with version 0 so that version 0 files can be converted to version 1?

Reply:
Yes, they could be loaded by setting the offsets to 0 and the sizes to the old gridSize. Didn't think of that and thought reloading the chunks would be enough. Should I do it?

Reply:
I always try to keep the files compatible so that the original chunks aren't required anymore once they have been loaded so that new things "just work" with new Chunky versions.
If you don't want to, that's fine too. It's something that could be added later on.